cypress/pageobjects/volume.po.ts

```diff
@@ -0,0 +1,67 @@
+import { Constants } from "@/constants/constants";
+import LabeledInputPo from '@/utils/components/labeled-input.po';
+
+const constants = new Constants();
+
+interface ValueInterface {
+  namespace?: string,
+  name?: string,
+  description?: string,
+  source?: string,
+  image?: string,
+  size?: string,
+}
+
+export class VolumePage {
+  name() {
+    return new LabeledInputPo('.namespace-select > .labeled-input', `:contains("Name")`)
+  }
+
+  description() {
+    return new LabeledInputPo('.labeled-input', `:contains("Description")`)
+  }
+
+  public goToList() {
+    // cy.get('.nav').contains('Volumes').click();
+    cy.visit(constants.volumePage)
+    cy.intercept('GET', '/v1/harvester/persistentvolumeclaims').as('goToVolumeList');
+    cy.wait('@goToVolumeList');
+    cy.url().should('eq', constants.volumePage)
+  }
+
+  public goToCreate() {
+    this.goToList();
+    cy.contains('Create').click();
+  }
+
+  public goToEdit(name: string) {
+    this.goToList();
+    cy.get('.search').type(name);
```

Comment:
The selector should be stored as a class member not selected just based on a CSS class

private search = '.search';
cy.get(this.search).type(name);

Reply:
done